keygen/keygen_network.c

```diff
@@ -77,6 +77,9 @@ keygen_network_register(struct register_internal * C)
 	/* Success! */
 	return (0);
 
+err2:
+	netpacket_close(NPC);
+	network_fini();
```

Comment:
Style question here: if `network_fini()` is going to be called in the error path inside `keygen_network_register()` [which makes sense], then what happens to the error labels?

Currently, it seems that `err0` is only used for `return (-1);` and possibly `bsdtar->return_value = 1`.  There's nothing in `STYLE` that enforces this, but I vaguely recall you telling me that.

Should I use
```
err1:
        network_fini();
err0:
        return (-1);
```

even if nothing ever calls `err0` ?  Or am I mistaken in `err0` only being for a `return` statement, and it's ok to call `network_fini()` inside `err0:` ?